db/migrate/20170725111950_change_enterprise_contact.rb

```diff
@@ -0,0 +1,18 @@
+class ChangeEnterpriseContact < ActiveRecord::Migration
+  def up
+    remove_columns :enterprises, :email, :contact
```

Comment:
Hey @Matt-Yorkley, I definitely think we still need the `contact` field. As per my comment [here](https://github.com/openfoodfoundation/openfoodnetwork/issues/1668#issuecomment-316572812) it contains a 'name' for use in both internal notifications from the OFN to the user and in confirmation emails from the OFN to customers.

If you need to use the name `contact` for the association that determines who receives notifications, could we just rename the existing contact field to `contact_name` rather than removing it?

Reply:
@Matt-Yorkley unfortunately this still stands. We definitely need to `contact` field to remain, and it's use to remain unchanged. If you need to rename it to `contact_name` that is fine, but everything related to this field that was pulled out in 54bf490e948f2a86291ac283023502a6841cb3af really needs to stay, and the removal of the field itself in 0de988a97c449c20ddb29060d87617d3684e55b7 needs to not happen.